### ISSUE TITLE
HACS 2.0 new integration add

### ISSUE THREAD

**Issue by rmiras00** [chrisc123/jebao_aqua-homeassistant#9]
with the new version of hacs 2.0, when i try to add the integration to my HA, HACS send the next messenger:  The version dd21d01 for this integration can not be used with HACS.

**Comment by VMartovsky** [comment]
Yes, the same problem. I would really like to try your integration!

**Comment by chrisc123** [comment]
I've added in a hacs.json needed for version 2.0 and this should work now.

**Comment by VMartovsky** [comment]
Thank you, installed via HACS.
The problem when installing the integration. It asks to enter the login and password for jebao. I enter it - it gives an error:
Failed to authenticate. Check you credentials.
I log in through the app using these details. I changed the password - it didn't help. I think it might be a server issue.

usaepapp.gizwits.com  usapi.gizwits.com
euaepapp.gizwits.com  euapi.gizwits.com
I tried it, still the same.

How can I find out which servers are being used?
Any thoughts?
